### DISCUSSION TITLE
How to understand VSCode debugger extension ` Provide rich IntelliSense and hover information for the debug configuration attributes used by the debugger. `?

### DISCUSSION THREAD

**Discussion by liaodalin19903** [microsoft/vscode-discussions#388] [Extension Development QnA] [answered]
When I read this document [debugging](https://code.visualstudio.com/api/extension-capabilities/overview#debugging) 

> Provide rich IntelliSense and hover information for the debug configuration attributes used by the debugger. 

Is there any snapshot for illustrating this statement? 

whether the hover information means this?
![image](https://user-images.githubusercontent.com/12727582/211134825-d69fc8b1-73e7-482b-8480-9f73fa66d65e.png)

how about the IntelliSense?

**Comment by isidorn** [comment, marked as answer]
Intellisense in launch.json and the hover when user hovers over the attributes inside the launch.sjon
